modules/extension/xsd/xsd-fes/src/main/java/org/geotools/filter/v2_0/bindings/BBOXTypeBinding.java

```diff
@@ -82,7 +82,7 @@ public Class getType() {
     public Object getProperty(Object object, QName name) throws Exception {
         BBOX box = (BBOX) object;
 
-        if (FES.ValueReference.equals(name)) {
+        if (FES.expression.equals(name)) {
```

Comment:
Interesting... but legit it seems, from the schema:

```
<xsd:complexType name="BBOXType">
<xsd:complexContent>
<xsd:extension base="fes:SpatialOpsType">
<xsd:choice maxOccurs="2">
<xsd:element ref="fes:expression"/>
<xsd:any namespace="##other"/>
</xsd:choice>
</xsd:extension>
</xsd:complexContent>
</xsd:complexType>
```
Guess the original writer just migrated the code from the superclass, without noticing the schema definition is quite different compared to Filter 1.1, e.g.:

```
<xsd:complexType name="BBOXType">
<xsd:complexContent>
<xsd:extension base="ogc:SpatialOpsType">
<xsd:sequence>
<xsd:element ref="ogc:PropertyName" minOccurs="0"/>
<xsd:element ref="gml:Envelope"/>
</xsd:sequence>
</xsd:extension>
</xsd:complexContent>
</xsd:complexType>
```